test/kubemark/pre-existing/README.md

```diff
@@ -0,0 +1,45 @@
+# Kubemark Pre-existing Provider Guide
+
+**Pre-existing Kubemark Master**
```

Comment:
If you want to reuse the kubernetes-master for the purpose of (2), I'm really against of making it explicit. I'm fine with making it possible configure (e.g. by setting KUBEMARK_MASTER=<address of kubernetes master> or sth like that), but I'm really against making this a silent assumption (since this is by far not the setup we recommend).